python/usb_console/simulation.py

```diff
@@ -53,18 +53,21 @@ def configure_sim(self):
         start_time = timeit.default_timer()
 
         self.eng = matlab.engine.start_matlab()
-        path1 = os.path.join(
+        path = os.path.join(
             os.path.dirname(os.path.abspath(__file__)), "../../MATLAB")
-        path2 = os.path.join(path1, "utl")
-        path3 = os.path.join(path1, "environmental_models")
-        path4 = os.path.join(path1, "environmental_models/helper_functions")
-        self.eng.addpath(path1, nargout=0)
-        self.eng.addpath(path2, nargout=0)
-        self.eng.addpath(path3, nargout=0)
-        self.eng.addpath(path4, nargout=0)
+        self.eng.addpath(path, nargout=0)
 
-        self.eng.eval("global const", nargout=0)
         self.eng.config(nargout=0)
+        self.eng.eval("global const", nargout=0)
+        self.eng.eval("global main_state_trajectory", nargout=0)
+        self.eng.eval("global computer_state_follower_trajectory", nargout=0)
+        self.eng.eval("global computer_state_leader_trajectory", nargout=0)
+
```

Comment:
You don't need these in python, they are just lists of the states for plotting at the end of the simulation.